addons/adminMenu/XEH_postInit.sqf

```diff
@@ -48,6 +48,11 @@
     [] call EFUNC(markers,initMarkerHash);
 }] call CBA_fnc_addEventHandler;
 
+[QGVAR(missionTestingAddAction), {
+     private _displayChecklist = ["displayChecklist", "Testing Menu", "", {[] call EFUNC(missionTesting,displayMenu)}, {true}, {}, [], [], 0] call ace_interact_menu_fnc_createAction;
+    ["CAManBase", 1, ["ACE_SelfActions"], _displayChecklist,true] call ace_interact_menu_fnc_addActionToClass;
```

Comment:
Space after `_displayChecklist`

Reply:
There was a space before the `_displayChecklist` is this the one that you mean or am I mission something?

Reply:
`["CAManBase", 1, ["ACE_SelfActions"], _displayChecklist,true]` -> `["CAManBase", 1, ["ACE_SelfActions"], _displayChecklist, true]`